client/app/components/form/MultiStepButtons.tsx

```diff
@@ -67,7 +66,7 @@ const SubmitButton: React.FunctionComponent<SubmitButtonProps> = ({
               type="button"
               disabled={isFinalStep || isSubmitting}
             >
-              Next
+              {isSubmitting ? "Save and Continue" : "Next"}
```

Comment:
necessary or else the button flashes between "Save And continue" and "Next" while submitting